lib/consul/async/consul_template_engine.rb

```diff
@@ -55,7 +56,7 @@ def run(template_manager)
               template_manager.terminate
               EventMachine.stop
             rescue StandardError => e
-              STDERR.puts "[ERROR] Fatal error occured: #{e.inspect} - #{e.backtrace}"
+              STDERR.puts "[ERROR] Fatal error occured: #{e.inspect} - #{e.backtrace.join("\n\t")}"
```

Comment:
Change this later, it is probably a good idea, but keep patch smallest as possible

Reply:
Roger